app/models/conversion_host.rb

```diff
@@ -274,16 +277,39 @@ def miq_ssh_util_args_manageiq_providers_openstack_cloudmanager_vm
   # +extra_vars+ option should be a hash of key/value pairs which, if present,
   # will be passed to the '-e' flag.
   #
-  def ansible_playbook(playbook, extra_vars = {})
-    command = "ansible-playbook #{playbook} -i #{ipaddress}"
+  def ansible_playbook(playbook, extra_vars = {}, auth_type = nil)
+    host = hostname || ipaddress
+
+    command = "ansible-playbook #{playbook} --inventory #{host}, --become -vvv"
+
+    auth = authentication_type(auth_type) || authentications.first
+    command += " --user #{auth.userid}"
```

Comment:
Minor, but prefer `<<` over `+=` for String building to avoid intermediate strings.

Reply:
Done